vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopIsoProtocolTest.kt

```diff
@@ -204,11 +199,11 @@ private suspend fun runProcess(
     holderSiop: OidcSiopWallet,
 ): IsoDocumentParsed {
     val authnRequest = verifierSiop.createAuthnRequestUrl(
-        walletUrl = walletUrl,
-        requestOptions = requestOptions
+        walletUrl = walletUrl, requestOptions = requestOptions
```

Comment:
Please keep the line breaks

Reply:
fixed in commit 1ce0f0f2b425074148a6b6972b355aafae3e034c